cmd/remoteva/main.go

```diff
@@ -18,6 +19,24 @@ import (
 type Config struct {
 	RVA struct {
 		vaConfig.Common
+
+		// VerifyGRPCClientCertIfGiven, when disabled, will cause the remoteva
+		// server (which receives gRPCs from a boulder-va client) to use our
+		// default RequireAndVerifyClientCert policy. When enabled, the remoteva
+		// server will instead use the less secure VerifyClientCertIfGiven
+		// policy. It should typically be used in conjunction with the
+		// boulder-va "RVATLSClient" configuration object.
+
+		// An operator may choose to enable this if the remoteva server is
```

Comment:
To keep the comment tied together in godoc
```suggestion
		// boulder-va "RVATLSClient" configuration object.
		//
		// An operator may choose to enable this if the remoteva server is
```